DevSkim-DotNet/Microsoft.DevSkim/Language.cs

```diff
@@ -61,7 +61,7 @@ public static string GetCommentInline(string language)
             {
                 foreach (Comment comment in Instance.Comments)
                 {
-                    if (comment.Languages.Contains(language.ToLower()) && comment.Inline is { })
+                    if (comment.Languages.Contains(language.ToLower(CultureInfo.InvariantCulture)) && comment.Inline is { })
```

Comment:
Aren't the language values static not a local culture impacted value?  Why would the value formatting be affected by the culture etc.

Reply:
In this case language is a parameter to a public methods so we actually have no guarantees at all about it. 

The recommended practice is to explicitly say which kind of culture info comparison you want to avoid potential unexpected behavior. you'll see this if you turn on Roslyn Analyzers.